### PR TITLE
added monitoring for the new hosts

### PR DESCRIPTION
**Pull request by adrianjgeorge**

### CODE REVIEW

**Comment by dmnjohns** [comment]
I wonder if, at some point, we could convert all of these nginx configuration files into Puppet using:
https://forge.puppet.com/puppet/nginx

**Comment by dmnjohns** [comment]
You probably want to add all of the new hosts to the host groups they belong to. That is, modify the `hostgroups.cfg` file.

File: myModules/repose_nagios/files/nagios_config/hostgroups.cfg [hunk 1]
```diff
@@ -9,7 +9,7 @@ define hostgroup {
 define hostgroup {
     hostgroup_name jenkins-slaves
     alias Jenkins Slave Servers
-    members slave1,slave2,slave3,slave4,slave5
+    members slave1,slave2,slave3,slave4,slave5,slave6,salve7,slave8,slave9
```
Comment (dmnjohns) [inline myModules/repose_nagios/files/nagios_config/hostgroups.cfg]: Typo. `salve`

**Comment by adrianjgeorge** [comment]
i did, just now
as for using the puppet thing for nagios, i suspect it's way more complicated then we want

**Comment by dmnjohns** [comment]
Side note, the `nginx-servers` and `performance-servers` host groups have the same alias. I feel like we could give `performance-servers` a better alias.

**Comment by dmnjohns** [comment]
Oh, and we probably want to add `grafana` to the `nginx-servers` host group as well. All of the HTTP checks are defined on that host group, and since Grafana is a HTTP service, I assume we want those checks on that host as well.

**Comment by adrianjgeorge** [comment]
yeah the alias should be changed, influx doesn't need http checks

**Comment by adrianjgeorge** [comment]
the nginx service is checking things that are not on on grafana

**Comment by adrianjgeorge** [comment]
including the https ports and the nginx process

**Comment by dmnjohns** [comment]
In that case, do we want to add a HTTP check specifically for Grafana?

**Comment by adrianjgeorge** [comment]
maybe? not trying to solve that problem right now. i'm just trying to get the new slaves on nagios

**Comment by dmnjohns** [comment]
That's fair, but I think that's something we missed when we were setting up Grafana in Nagios. I figured it would be easy enough to knock out here, but we could create a story or something for that if you would prefer.

Depending on your answer to that, I'm just waiting for the typo to be fixed, then I'll merge this.

**Comment by adrianjgeorge** [comment]
It's not going to be quite that simple

**Comment by adrianjgeorge** [comment]
lol, i just sorted it and added it

**Pull request merged**